cogs/displayinfo.py

```diff
@@ -0,0 +1,131 @@
+import discord
+import constantvariables
+from discord.ext import commands
+from discord.ext import tasks
+from datetime import datetime
+from datetime import date
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+import asyncio
+
+
+# MongoDB URI for connecting to the database
+uri = "mongodb+srv://user:user@rasenbotcluster.toeujwx.mongodb.net/?retryWrites=true&w=majority"
+
+# Create a new client and connect to the MongoDB server
+mongoClient = MongoClient(uri, server_api=ServerApi('1'))
+
+
+try:
+    # Send a ping to confirm a successful connection to MongoDB
+    mongoClient.admin.command('ping')
+    print("Pinged your deployment. You successfully connected to MongoDB!")
+except Exception as e:
+    print(e)
+
+# Select the database
+db = mongoClient.Rasenbot
+
+
+
+class DisplayInfo(commands.Cog):
+
+    def __init__(self,client):
+        self.client = client
+        self.timers = {}
+        # self.tracked_games = ['Rocket League', 'League of Legends', "Spotify"]
+        self.current_collection = ""
+        self.count = 0
+        self.current_date = str(date.today())
+
+    
+    # @commands.Cog.listener()
+    # async def on_ready(self):
+    #     for guild in self.client.guilds:
+    #         guild_name = guild.name
+
+    #         # Check if the MongoDB collection for the guild already exists
+    #         if guild_name not in db.list_collection_names():
+    #             # If the collection doesn't exist, create a new one for the guild
+    #             newCollection = db[guild_name]
+    #             self.current_collection = newCollection
+    #             newCollection.insert_one({})  # Insert a dummy document to initialize the collection
+    #             self.timers = self.current_collection.find_one()
+    #         else:
+    #             # If the collection exists, get the existing collection for the guild
+    #             self.current_collection = db[guild_name]
+    #             # Load existing timers (if any) from the MongoDB collection
+    #             self.timers = self.current_collection.find_one()
+    
+
+
+    # The 'before' parameter represents the member's previous state, and 'after' represents the current state.
+    @commands.Cog.listener()
+    async def on_member_update(self, before:discord.Member, after:discord.Member):
```

Comment:
Changing `before` and `after` to something that includes `member` might be better for readability even though the types are listed. Something like `mem_before` or `user_before`.